### PR TITLE
Update Personal Capital

### PR DESCRIPTION
**Pull request by max-rousseau**
Personal Capital also supports sms & totp now. The documentation link proposed is also more pertinent.

### CODE REVIEW

File: entries/p/personalcapital.com.json [hunk 1]
```diff
@@ -4,11 +4,13 @@
     "url": "https://www.personalcapital.com",
     "tfa": [
       "call",
-      "email"
+      "email",
+      "sms",
+      "totp"
```
Comment (Carlgo11) [inline entries/p/personalcapital.com.json]: According to https://support.personalcapital.com/hc/en-us/articles/360056729413 it only works with Personal Capital app. Can you confirm this?

Comment (max-rousseau) [reply entries/p/personalcapital.com.json]: It does work with the Personal Capital app using either Push flow or totp code. The generated QR code shown in enrollment however is RFC compliant and can be also registered with any totp app.

Comment (Carlgo11) [reply entries/p/personalcapital.com.json]: Okay so it sounds like both `totp` and `custom-software` should be set.
See [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties) for more info.

Comment (max-rousseau) [reply entries/p/personalcapital.com.json]: Fair point, pushed update.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
still needs the custom-software tag with `Personal Capital App` value.

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**